stdlib/list.mli

```diff
@@ -73,6 +73,11 @@ val nth_opt: 'a list -> int -> 'a option
 val rev : 'a list -> 'a list
 (** List reversal. *)
 
+val init : int -> (int -> 'a) -> 'a list
+(** [List.init len f] is [f 0; f 1; ...; f (n-1)].
```

Comment:
This comment is now longer true - the clearest is probably to add ` (evaluated left to right)`

The `@since` and `@raise` changes still need making.